src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3762,6 +3762,9 @@ void SpellMgr::LoadSpellInfoCorrections()
                 break;
             // ENDOF ISLE OF CONQUEST SPELLS
             //
+            case 34905:
```

Comment:
ha, I am dumb, did not even see that in list of actions. Thanks. Will change and test.